src/slideseq/plot/plot_downsampling.py

```diff
@@ -11,12 +12,14 @@
 
 from slideseq.plot import read_dge_summary
 
+import matplotlib.pyplot as plt
+
+from matplotlib.offsetbox import AnchoredText
+
 log = logging.getLogger(__name__)
 
 
-def plot_downsampling(
-    downsampling_output: list[tuple[float, Path]], matched_path: Path, figure_path: Path
-):
+def plot_downsampling(downsampling_output: list[tuple[float, Path]], matched_path: Path, figure_path: Path):
```

Comment:
Should stick with the other formatting, this line is too long
```suggestion
def plot_downsampling(
    downsampling_output: list[tuple[float, Path]], matched_path: Path, figure_path: Path
):
```